postgres/codenamemap.yaml

```diff
@@ -40,13 +46,19 @@
   {% if repo.use_upstream_repo == true %}
     {% set fromrepo = repo.fromrepo|default(name ~ '-pgdg', true) %}
```

Comment:
I guess I missed this before, but the `fromrepo` for RedHat family should be equal to `pkg_repo.name` value (`pgdg{{ release }}`). This need to be corrected.